styles/widgets/generic/gridBase.generic.less

```diff
@@ -720,16 +720,24 @@
                 outline: none;
             }
 
-            &.dx-data-row:not(.dx-edit-row) {
-                & > td:not(.dx-focused),
-                & > tr > td:not(.dx-focused),
-                & .dx-command-edit:not(.dx-focused) .dx-link {
+            &.dx-data-row {
+                & > td,
+                & > tr > td,
+                & .dx-command-edit {
                     background-color: ~"@{@{widgetName}-row-focused-bg}";
                     color: ~"@{@{widgetName}-row-focused-color}";
 
-                    & .dx-@{widgetName}-group-opened,
-                    & .dx-@{widgetName}-group-closed {
+                    input {
+                        border-radius: 0;
+                    }
+
+                    .dx-link {
                         color: ~"@{@{widgetName}-row-focused-color}";
+
+                        & .dx-@{widgetName}-group-opened,
+                        & .dx-@{widgetName}-group-closed {
+                            color: ~"@{@{widgetName}-row-focused-color}";
```

Comment:
Is this block of styles correct? The .dx-@{widgetName}-group-opened and .the dx-@{widgetName}-group-closed classes can not be inside .dx-link